paper-input-error.html

```diff
@@ -63,19 +63,11 @@
     properties: {
 
       /**
-       * The associated input element.
+       * Set to true to show the error.
        */
-      inputElement: {
-        observer: '_inputElementChanged',
-        type: Object
-      },
-
-      /**
-       * The current value of the input element.
-       */
-      value: {
-        observer: '_valueChanged',
-        type: String
+      invalid: {
```

Comment:
Just an idea, but this attribute falls into the category of attributes I wish were reflected when I'm trying to debug things by scanning the Elements tab in inspector.